polyfills/HTMLCanvasElement/protoype/toBlob/config.json

```diff
@@ -0,0 +1,11 @@
+{
+	"browsers": {
+		"chrome": "<50",
+		"firefox": "<19",
+		"safari": "*"
```

Comment:
Could we also add other browsers to this list such as:
- Opera
- IE
- iOS
- Android